extensions/eclipse-che-theia-plugin-ext/src/common/che-protocol.ts

```diff
@@ -9,7 +9,7 @@
  **********************************************************************/
 
 import { ProxyIdentifier, createProxyIdentifier } from '@theia/plugin-ext/lib/api/rpc-protocol';
-import * as che from '@eclipse-che/plugin';
+import { che as api } from '@eclipse-che/api';
```

Comment:
hello, is there a reason to rename che to api ?

Reply:
There is che in '@eclipse-che/plugin' and che in '@eclipse-che/api'. We have to separate them somehow.

Reply:
Evhen and me have discussed this and decided that renaming che to api when importing from '@eclipse-che/api' is a variant.

Reply:
chePlugin and cheApi ?

IMHO api is too generic when reading the code

Reply:
```
import * as chePlugin from '@eclipse-che/plugin';
import { che as cheApi } from '@eclipse-che/api';
```

Reply:
maybe it's verbose but it's easy to track in the code (to avoid also mistakes)

Reply:
```typescript
import * as chePlugin from '@eclipse-che/plugin';
import { che as cheApi } from '@eclipse-che/api';
```
Is fine for me.

Reply:
Reworked